docs/variants/clevo_nv41/flashing_internal.md

```diff
@@ -50,18 +50,18 @@ following command:
 To flash coreboot to the laptop, execute the following command - replace [path]
 with the path to the coreboot image you want to flash, e.g. `build/coreboot.rom`.
 
-If stock firmware or coreboot version below v0.5.0 is currently installed:
+If stock firmware or coreboot version below v1.1.0 is currently installed:
```

Comment:
@mkopec Maybe it should be `Dasharo version` ?

Reply:
Yes. Changed most occurences of coreboot to Dasharo